test/schemaValidation.test.ts

```diff
@@ -1560,6 +1560,15 @@ obj:
   });
 
   describe('Bug fixes', () => {
+    it('should handle "additionalItems" not schema or boolean', async () => {
```

Comment:
Is this an accurate description of the test?

Reply:
No, it wasn’t